Stateless/project.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "2.5.0-*",
+  "dependencies": {
+    "NETStandard.Library": "1.6.0"
+  },
+  "packOptions": {
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+    "projectUrl": "https://github.com/nblumhardt/stateless",
+    "iconUrl": "https://storage.googleapis.com/google-code-archive/v2/code.google.com/stateless/logo.png",
```

Comment:
Exactly, cause nowadays there is no log in nuget package (just nuget default one).